Web-Programming/OpenWeatherMaps.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch')
```

Comment:
this file is already in this #196 PR, please remove this  file from this PR  so, no conflict arises

Reply:
I branched off from the web-programming branch by mistake and due to the HeapSort issue I am unable to rebase the branch on the master. What should I do?